src/components/LastInteractions.js

```diff
@@ -25,6 +25,9 @@ const useStyles = makeStyles((theme) => ({
     textDecoration: 'none',
     color: theme.palette.text.primary,
   },
+  link: {
+    position: 'relative',
```

Comment:
without it there is no names below images of people on dashboard 

> Great fix! I have some concerns in the comments below. And also I was wondering if it maters that the checkboxed circle is inverted in colors:
> 
> ![Screenshot from 2022-06-17 18-35-14](https://user-images.githubusercontent.com/20553836/174495687-c8763563-b8f4-4c36-affa-686dd61d9084.png)

We inverted it because as it is mui component it was impossible to style it. But now I am thinking that we removed gradient as well so I will have a look into that again as it might bring solution.